code/modules/mob/living/simple_animal/abnormality/_abnormality.dm

```diff
@@ -398,13 +398,17 @@
 
 // Actions
 /datum/action/innate/abnormality_attack
-	name = "Megafauna Attack"
-	icon_icon = 'icons/mob/actions/actions_animal.dmi'
+	name = "Abnormality Attack"
+	icon_icon = 'icons/mob/actions/actions_abnormality.dmi'
 	button_icon_state = ""
 	var/mob/living/simple_animal/hostile/abnormality/A
 	var/chosen_message
 	var/chosen_attack_num = 0
 
+/datum/action/innate/abnormality_attack/Trigger()
+	if(A.IsContained()) // No more using cooldowns while contained
+		return FALSE
+
```

Comment:
```suggestion
/datum/action/innate/abnormality_attack/Trigger()
	if(A.IsContained()) // No more using cooldowns while contained
		return FALSE
	return ..()
```
This relies on calling it's parent procs.

Reply:
Fixed now, did an stupid oops, that was supposed to be removed actually since it doesnt work, so i put the IsContained inside the buttons instead and removed this.